index.html

```diff
@@ -219,6 +219,20 @@ <h2>About
 
         </div>
     </section>
+
+    <section class="contact">
+        <h2 class="contact-title">Contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that
+            needs coding. I’d love to help with it </p>
+        <form class="contact-form" action="https://formspree.io/f/myyogbyl" method="POST">
+            <input class="form-input" name="name" maxlength="30" placeholder="Name" type="text" required />
+            <input class="form-input" name="email" placeholder="email@example.com" type="email" required />
+            <textarea maxlength="500" name="message" rows="6" cols="10" class="form-input"
+                placeholder="Write your message here" required></textarea>
+            <button type="submit" class="contact-btn">Get It Touch</button>
```

Comment:
- [ ] Good job implementing interactions to this submit button but one thing that can improve the hover interaction is to add the pointer on the button.